src/documentation/how-to-write-documentation.md

```diff
@@ -0,0 +1,149 @@
+# How to write documentation
+
+This document explains how to write documentation for the std/core public APIs.
+
+Let's start with some general information:
+
+### Contractions
+
+It is common in english to have contractions such "don't" or "can't". They can
+never be used in documentation. Always write their "full form":
+
+ * "do not" instead of "don't"
+ * "cannot" instead of "can't"
+ * "it would" instead of "it'd"
+ * "it will" instead of "it'll"
+ * "you are" instead of "you're"
+ * "he is"/"he has" instead of "he's"
```

Comment:
```suggestion
 * "they are" instead of "they're"
```

(Let's steer people towards the kinds of examples that should appear in documentation.)